packages/fx-core/src/common/cpUtils.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { LogProvider } from "@microsoft/teamsfx-api";
+import * as cp from "child_process";
+import * as os from "os";
+
+export async function executeCommand(
+  workingDirectory: string | undefined,
```

Comment:
can we adjust the sequence of parameters and give default value for some parameters? I saw you set some undefined value in arm implementation. maybe we can try to reduce the need of setting undefined value as parameter (not total remove the requirement).
For example, workingDirectory may put to latter position and set undefined as default value because it's not always required.